lib/oli_web/components/delivery/actions/actions.ex

```diff
@@ -92,10 +105,35 @@ defmodule OliWeb.Components.Delivery.Actions do
               </button>
           </div>
         {/if}
+
+        <div class="ml-auto">
+          <button phx-click={JS.push("open", target: "#unenroll_user_modal")} class="btn btn-danger">Unenroll</button>
+        </div>
```

Comment:
Looks good! But... What do you think about adding some text on the left side of the button so that it is not loose on the right side? 
I'm thinking of some text like the one that has the functionality to change the role of the user above the unenroll button.